astropy/coordinates/tests/test_name_resolve.py

```diff
@@ -114,10 +113,8 @@ def test_names():
     # "all" choice should ask for SIMBAD, then NED, then Vizier: "SNV" in the url
     with pytest.raises(
         NameResolveError,
-        match=re.escape(
-            "Unable to find coordinates for name 'm87h34hhh' "
-            "using https://cds.unistra.fr/cgi-bin/nph-sesame/SNV?m87h34hhh"
-        ),
+        # avoid hard-coding an exact url as it might depend on external state
+        match="Unable to find coordinates for name 'm87h34hhh' using http",
```

Comment:
According to the comment above, we should check for "SNV" in the url, so somehting like `...http.*sesame/SNV?m87h34hhh`

Reply:
done